site/proxy.md

```diff
@@ -178,6 +181,18 @@ Note how regexp substitution groups should be prepended with a dollar sign
 (e.g. `$1`). For further details on the regular expression syntax please see
 [Google's re2 documentation](https://github.com/google/re2/wiki/Syntax).
 
+Similarly, if we launch the proxy using `--hostname-from-label hostname-label`
+
+    host1$ weave launch-router && weave launch-proxy --hostname-from-label hostname-label
+    host1$ eval "$(weave env)"
+
+the hostname will be extracted from the label with key `hostname-label`
+
+    host1$ docker run -ti --label=hostname-label=my-app-foo ubuntu ping my-app-foo
+    PING my-app-foo.weave.local (10.32.0.2) 56(84) bytes of data.
+    64 bytes from my-app-foo.weave.local (10.32.0.2): icmp_seq=1 ttl=64 time=0.031 ms
+    64 bytes from my-app-foo.weave.local (10.32.0.2): icmp_seq=2 ttl=64 time=0.042 ms
+
```

Comment:
> I would like to show both how it works with labels and how the container name is used if no label is provided.

And my point was that I don't think that is necessary. Just show all three being used together.